app/views/candidate_interface/personal_statement/show.html.erb

```diff
@@ -4,9 +4,19 @@
 <%= form_with model: @section_complete_form, url: candidate_interface_becoming_a_teacher_complete_path, method: :patch do |f| %>
   <%= f.govuk_error_summary %>
 
-  <h1 class="govuk-heading-xl">
-    <%= t('page_titles.becoming_a_teacher') %>
-  </h1>
+  <% if FeatureFlag.active?(:one_personal_statement) && @application_form.single_personal_statement? %>
+    <h1 class="govuk-heading-xl">
+      <%= t('page_titles.personal_statement') %>
+      <% if @application_form.subject_knowledge.nil? && @application_form&.previous_application_form&.subject_knowledge.present? %>
+        <p class="govuk-body govuk-!-margin-top-2 govuk-!-margin-bottom-0">Your personal statement has been combined into one statement instead of 2.</p>
```

Comment:
```suggestion
        <p class="govuk-body govuk-!-margin-top-2 govuk-!-margin-bottom-0">Your personal statement has been combined into one statement instead of two.</p>
```